src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -169,6 +177,35 @@ func useAllCores() {
 	runtime.GOMAXPROCS(numcores)
 }
 
+func configureLogging() *rotator.SizeRotator {
+	logdir := appdir.Logs("Lantern")
+	log.Debugf("Placing logs in %v", logdir)
+	if _, err := os.Stat(logdir); err != nil {
+		if os.IsNotExist(err) {
+			// Create log dir
+			if err := os.MkdirAll(logdir, 0755); err != nil {
+				log.Fatalf("Unable to create logdir at %s: %s", logdir, err)
+			}
+		}
+	}
+	file := rotator.NewSizeRotator(filepath.Join(logdir, "lantern.log"))
+	// Set log files to 1 MB
+	file.RotationSize = 1 * 1024 * 1024
+	// Keep up to 20 log files
+	file.MaxRotation = 20
+	errorOut := timestamped(io.MultiWriter(file, os.Stderr))
```

Comment:
MultiWriter will stop writing on first error, we'd better to write to Stderr first.